Detectors/TRD/workflow/src/TRDTrapSimulatorSpec.cxx

```diff
@@ -301,6 +302,27 @@ bool digitindexcompare(unsigned int A, unsigned int B, const std::vector<o2::trd
     return 0;
 }
 
+void TRDDPLTrapSimulatorTask::fixTriggerRecords(std::vector<o2::trd::TriggerRecord>& trigRecord)
```

Comment:
The initial TriggerRecord could in principle by zero and still they are shifted, right?

Reply:
@martenole  I dont follow?

Reply:
Let's say the TriggerRecords are shifted by one. But by chance the first event actually contains zero tracklets. Then the fix would still be required but I would only get a warning.

Reply:
the zero zero check is on the digits triggerrecord not the tracklet triggerrecord.

Reply:
Ok, I did not see that. Fine with me then